knowledge-base/grid-column-in-custom-component-is-last.md

```diff
@@ -107,6 +107,7 @@ You can observe and reproduce the same behavior with the example below.
 }
 ````
 
+>tip If you want to have conditional columns, you can use the [`Visible`]({%slug grid-columns-visible%}) parameter. With a certain condition, for example, a ternary operator or a method that returns a bool.
```

Comment:
The second sentence follows Bulgarian grammar rules (no subject, no action), but that approach is not used in English.

```suggestion
>tip If you want to have conditional columns, you can use the [`Visible`]({%slug grid-columns-visible%}) parameter. The condition can be a ternary operator or a method that returns a bool.
```